ansible/README.md

```diff
@@ -0,0 +1,151 @@
+# Ansible playbook for Tendermint
+
+![Ansible plus Tendermint](img/a_plus_t.png)
+
+* [Prerequisites](#Prerequisites)
+* [Ansible setup](#Ansible setup)
+* [Running the playbook](#Running the playbook)
+* [Example playbook that configures a Tendermint on Ubuntu](#example-playbook-that-configures-a-tendermint-on-ubuntu)
+
+The playbooks in this folder run [ansible](http://www.ansible.com/) roles which:
+
+* install tendermint
+* install basecoin
+* configure tendermint and basecoin
+* start/stop tendermint and basecoin and reset their configuration
+
+## Prerequisites
+
+* Ansible 2.0 or higher
+* SSH key to the servers
+
+Optional for DigitalOcean droplets:
+* DigitalOcean API Token
+* python dopy package
+
+Head over to the [Terraform folder](https://github.com/tendermint/tools) for a description on how to get a DigitalOcean API Token.
+
+Optional for Amazon AWS instances:
+* Amazon AWS API access key ID and secret access key.
+
+The cloud inventory scripts come from the ansible team at their [GitHub](https://github.com/ansible/ansible) page. You can get the latest version from the contrib/inventory folder.
+
+## Ansible setup
```

Comment:
Ok, Ansible installation document clarified. I'm still working on separating tendermint and basecoin (and adding ethermint), it should be done sometime early next week.